configurations/sync_pool_kenya_demogs_to_text_it.py

```diff
@@ -0,0 +1,74 @@
+        credentials_file_url="gs://avf-credentials/avf-engagement-databases-firebase-credentials-file.json",
+        database_path="engagement_databases/POOL-KENYA"
+    ),
+    uuid_table=UUIDTableClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+        table_name="avf-global-urn-to-participant-uuid",
+        uuid_prefix="avf-participant-uuid-"
+    ),
+    operations_dashboard=OperationsDashboardConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json",
+    ),
+    rapid_pro_target=RapidProTarget(
+        rapid_pro=RapidProClientConfiguration(
+            domain="textit.com",
+            token_file_url="gs://avf-credentials/pool-kenya-textit-token.txt"
+        ),
+        sync_config=EngagementDBToRapidProConfiguration(
+            normal_datasets=[
+                DatasetConfiguration(
+                    engagement_db_datasets=["age"],
+                    rapid_pro_contact_field=ContactField(key="pool_kenya_age", label="pool kenya age")
+                ),
+                DatasetConfiguration(
+                    engagement_db_datasets=["gender"],
+                    rapid_pro_contact_field=ContactField(key="pool_kenya_gender", label="pool kenya gender")
+                ),
+                DatasetConfiguration(
+                    engagement_db_datasets=["location"],
+                    rapid_pro_contact_field=ContactField(key="pool_kenya_location", label="pool kenya location")
+                ),
+                DatasetConfiguration(
+                    engagement_db_datasets=["disabled"],
+                    rapid_pro_contact_field=ContactField(key="pool_kenya_disabled", label="pool kenya disabled")
+                ),
+            ],
+            consent_withdrawn_dataset=DatasetConfiguration(
+                engagement_db_datasets=["age", "gender", "location", "disabled", "preferred_language", # Demogs
+                                        "machakos_governor_poll_2022", "machakos_senator_poll_2022", "machakos_womenrep_poll_2022_rita", 
+                                        "machakos_womenrep_poll_2022", "machakos_county_priorities_2022", "icl_pool_invitation_2022", 
+                                        "machakos_poll_registration_2022", # ICL datasets
+                                        "aik_household_income", "aik_communities", "aik_religion", "aik_employment_status", "aik_education", 
+                                        "aik_pool_invitation_2022", "aik_voting_participation", "aik_indigenous_or_minority", "aik_sexual_assault",
+                                        "aik_violence_displacement", "aik_vandalism_theft_incidents", "aik_inability_to_work", "aik_unsafe_areas",
+                                        "aik_political_events_disruption", "aik_political_environment", "aik_electoral_violence_anxiety",
```

Comment:
I think
```
# Demogs:
"age", "gender", ...
# ICL Datasets:
"machakos_governor_poll_2022", ...
```
would be slightly clearer than trailing comments.

In future we can think about how to make this work without needing an ever-growing list of datasets to be set here.

Reply:
Noted.